components/DropdownLight/DropdownLight.unit.test.jsx

```diff
@@ -81,4 +85,37 @@ describe('<DropdownLight />', () => {
 
     expect(input.name).toMatch(INPUT_NAME);
   });
+
+  it('should close Dropdown Options when user press Escape', () => {
+    render(<DropdownLight items={itemsStringMock} />);
+
+    const dropdown = screen.getByRole('button');
+    userEvent.click(dropdown);
+
+    userEvent.keyboard(`${EscapeKeyCode}`);
+
+    expect(screen.queryAllByRole('option')).toHaveLength(0);
+
+    expect(
+      screen.getByRole('button', { name: 'abrir lista de itens' }),
+    ).toBeInTheDocument();
+  });
+
+  it('using only keyboard', () => {
+    render(<DropdownLight items={itemsStringMock} />);
+
+    userEvent.tab();
+    userEvent.keyboard(`${EnterKeyCode}`);
+
+    const bananaItem = itemsStringMock[1];
+
+    const input = screen.getByRole('textbox', { hidden: true });
+
+    userEvent.keyboard(
+      `${ArrowDownKeyCode}${ArrowDownKeyCode}${ArrowUpKeyCode}${EnterKeyCode}`,
+    );
```

Comment:
```suggestion
    userEvent.keyboard(
      `${ArrowDownKeyCode}${ArrowDownKeyCode}${ArrowUpKeyCode}${EnterKeyCode}`,
    );
    const bananaItem = itemsStringMock[1];
    const input = screen.getByRole('textbox', { hidden: true });
```

Reply:
Why should I return a list if I'm simulating a keyboard selection?